pkg/dashboard/api.go

```diff
@@ -99,13 +101,27 @@ func NewRouter(abortWeb context.CancelFunc, data *objects.DataLayer, debug bool)
 		api.Use(allowCORS)
 	}
 
-	configureStatic(api)
-	configureRoutes(abortWeb, data, api)
+	// Determine base path prefix for mounting the app under a subpath
```

Comment:
see my comment in server.go - the code would be simpler if the base path value is just passed to `NewRouter()`